.github/workflows/ci.yml

```diff
@@ -13,13 +13,13 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - '3.7'
           - '3.8'
           - '3.9'
           - '3.10'
           - '3.11'
-          - 'pypy-3.7'
+          - '3.12.0-beta.3'
```

Comment:
AFAIU, this would require updating the environment matrix in tox.ini as well?

Reply:
@n1ngu Just did the changes in https://github.com/tarekziade/molotov/pull/153/commits/5868c4600fda28a3e442cbb8770ebbf58578fd7f

Please approve the actions run so we can see if there are some actual failure in the Actions.